webcompat/api/uploads.py

```diff
@@ -26,11 +23,71 @@
 uploads = Blueprint('uploads', __name__, url_prefix='/upload',
                     template_folder='../templates')
 JSON_MIME = 'application/json'
-images = UploadSet('uploads', IMAGES)
-configure_uploads(app, images)
+ALLOWED_FORMATS = ('jpg', 'jpeg', 'jpe', 'png', 'gif', 'bmp')
 
-# limit image uploads to 4MB
-patch_request_class(app, 4 * 1024 * 1024)
+
+class UploadNotAllowed(Exception):
+    """
+    This exception is raised when we try to save a file that is not
+    allowed. Allowed files are defined in ALLOWED_FORMATS
+    """
```

Comment:
I guess I can fix it later. but I would rather have something slightly different here. Probably something derived from TypeError. But your code will work anyway here. Just a reminder for me.